includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -57,9 +57,9 @@
 		 * @access private
 		 */
 		private function __construct() {
+			self::$per_page  = absint( ! empty( $_GET['per_page'] ) ? $_GET['per_page'] : self::$per_page );
```

Comment:
@raftaar1191 Can you explain `absint( ! empty( $_GET['per_page'] )`

Reply:
Got it @ravinder
What you are pointing 
self::$per_page  = absint( ! empty( $_GET['per_page'] ) ? $_GET['per_page'] : self::$per_page );

I am using absint every time without depending on the situation 
Like if  $_GET['per_page'] is empty then I do not need to check self::$per_page as it datatype will always be INT

Reply:
Fixing that

Reply:
Update the code 

Thanks @ravinderk for pointing this out to me.